sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -360,15 +364,16 @@ async def mgmt_request_async(self, message, **kwargs):
         operation_type = kwargs.pop("operation_type", None)
         node = kwargs.pop("node", "$management")
         timeout = kwargs.pop('timeout', 0)
-        try:
-            mgmt_link = self._mgmt_links[node]
-        except KeyError:
-            mgmt_link = ManagementOperation(self._session, endpoint=node, **kwargs)
-            self._mgmt_links[node] = mgmt_link
-            await mgmt_link.open()
+        with self._lock:
```

Comment:
lock to prevent context switching when trying to create a mgmt_link -- uamqp uses the 1 mgmt lock for all auto-lock renewers,pyamqp was trying (and failing) to make multiple